dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -0,0 +1,2094 @@
+                                     .contentType(contentType))
+                        .andExpect(status().isNotFound());
+
+    }
+
+    @Test
+    public void postCollectionDefaultBitstreamReadGroupCreateDefaultBitstreamReadGroupUnProcessableName()
+        throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        GroupRest groupRest = new GroupRest();
+        groupRest.setName("Fail");
+        MetadataRest metadataRest = new MetadataRest();
+        metadataRest.put("dc.description", new MetadataValueRest("testingDescription"));
+        metadataRest.put("dc.subject", new MetadataValueRest("testSubject"));
+
+        groupRest.setMetadata(metadataRest);
+
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(post("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup")
+                                     .content(mapper.writeValueAsBytes(groupRest))
+                                     .contentType(contentType))
+                        .andExpect(status().isUnprocessableEntity());
+
+    }
+
+    @Test
+    public void postCollectionDefaultBitstreamReadGroupCreateDefaultBitstreamReadGroupUnProcessablePermanent()
+        throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        GroupRest groupRest = new GroupRest();
+        groupRest.setPermanent(true);
+        MetadataRest metadataRest = new MetadataRest();
+        metadataRest.put("dc.description", new MetadataValueRest("testingDescription"));
+        metadataRest.put("dc.subject", new MetadataValueRest("testSubject"));
+
+        groupRest.setMetadata(metadataRest);
+
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(post("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup")
+                                     .content(mapper.writeValueAsBytes(groupRest))
+                                     .contentType(contentType))
+                        .andExpect(status().isUnprocessableEntity());
+
+    }
+
+    @Test
+    public void deleteCollectionDefaultBitstreamReadGroupTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        String bitstreamGroupString = "BITSTREAM";
+        int defaultBitstreamRead = Constants.DEFAULT_BITSTREAM_READ;
+
+        Group role = createDefaultReadGroup(bitstreamGroupString, defaultBitstreamRead);
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup"))
+                        .andExpect(status().isNoContent());
+
+        Group anon = groupService.findByName(context, Group.ANONYMOUS);
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/bitstreamReadGroup"))
+                        .andExpect(status().isOk())
```

Comment:
please add a comment to note that deleting a custom default read group is expected to restore the default to anonymous and also make a PR for the REST Contract to note that (I'm fine with this behavior we only need to have it documented)